pkg/kubectl/cmd/util/cmdtools.go

```diff
@@ -111,53 +125,87 @@ func (e Editor) args(path string) []string {
 
 // Launch opens the described or returns an error. The TTY will be protected, and
 // SIGQUIT, SIGTERM, and SIGINT will all be trapped.
-func (e Editor) Launch(path string) error {
-	if len(e.Args) == 0 {
-		return fmt.Errorf("no editor defined, can't open %s", path)
-	}
-	abs, err := filepath.Abs(path)
-	if err != nil {
-		return err
+func (t CmdTool) Launch(paths []string) error {
+	args := make([]string, len(t.Args))
+	copy(args, t.Args)
```

Comment:
I thought those two are equivalent 
```
args := make([]string, len(t.Args))
copy(args, t.Args)
```
```
args := t.Args
```
just follow the same style [here](https://github.com/kubernetes/kubernetes/pull/46560/files#diff-3266a8985ae8c90ba82083a7fa12f10eR115), CMIIW